instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/AwsSdkRequest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2;
+
+import static io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdkRequestType.DynamoDB;
+import static io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdkRequestType.Kinesis;
+import static io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdkRequestType.S3;
+import static io.opentelemetry.instrumentation.awssdk.v2_2.AwsSdkRequestType.SQS;
+import static io.opentelemetry.instrumentation.awssdk.v2_2.FieldMapping.of;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import software.amazon.awssdk.core.SdkRequest;
+
+/**
+ * Temporary solution - maps only DynamoDB attributes. Final solution should be generated from AWS
+ * SDK automatically
+ * (https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/2291).
+ */
+public enum AwsSdkRequest {
+  // generic requests
+  DynamoDbRequest(DynamoDB, "DynamoDbRequest"),
+  S3Request(S3, "S3Request"),
+  SqsRequest(SQS, "SqsRequest"),
+  KinesisRequest(Kinesis, "KinesisRequest"),
+  // specific requests
+  CreateTable(
+      DynamoDB,
+      "CreateTableRequest",
+      of("awssdk.global_secondary_indexes", "globalSecondaryIndexes"),
```

Comment:
Thanks for adapting my initial proposal :) Can you update these with what I've set in this latest PR?

https://github.com/open-telemetry/opentelemetry-specification/pull/1422

Notably, `aws.dyanamodb.*` instead of `awssdk.*` (we can dream that the server will populate these some day right? :)

Reply:
Done :)